### ISSUE TITLE
Qwik City SSG - can't have special characters in dynamic route URI

### ISSUE THREAD

**Issue by locriacyber** [BuilderIO/qwik#1999]
### Qwik Version

A

### Operating System (or Browser)

N/A

### Which component is affected?

Qwik City

### Expected Behaviour

When generating route like `/[abc]`, when `abc` has space (0x20) in it, it should retain the space character.

For example, if the URI is `/a%20b`, the generated file should be at `/dist/a b/index.html`

### Actual Behaviour

If the URI is `/a%20b`, the generated file is at `/dist/a%20b/index.html`

A lot of HTTP server expect the file name not to be encoded.

**Comment by manucorporat** [comment]
This makes sense! wanna give it a try trying to fix this issue? hopefully it's a simple fix

**Comment by gioboa** [comment]
@iacore Hi, did you solve your problem?
I looked into the code and there is a specific test that verify the space case. https://github.com/BuilderIO/qwik/blob/341038bb419ffd0969cd48aa6cf303217bedacf7/packages/qwik-city/utils/pathname.unit.ts#L56

**Comment by iacore** [comment]
https://github.com/BuilderIO/qwik/blob/341038bb419ffd0969cd48aa6cf303217bedacf7/packages/qwik-city/utils/pathname.unit.ts#L59

**Comment by gioboa** [comment]
Yep, so how did you solve the issue? Which workaround did you use?

**Comment by iacore** [comment]
> Hi, did you solve your problem?

No.

**Comment by gioboa** [comment]
Which HTTP server are you using?

**Comment by iacore** [comment]
> Which HTTP server are you using?

`python -m http.server`
